tests/ops/api/v1/endpoints/test_privacy_request_endpoints.py

```diff
@@ -1628,6 +1663,49 @@ def test_approve_privacy_request(
 
         privacy_request.delete(db)
 
+    @mock.patch(
+        "fidesops.service.privacy_request.request_runner_service.run_privacy_request.delay"
+    )
+    def test_approve_privacy_request_creates_audit_log(
+        self,
+        submit_mock,
+        db,
+        url,
+        api_client,
+        generate_auth_header,
+        user,
+        privacy_request,
+    ):
+        privacy_request.status = PrivacyRequestStatus.pending
```

Comment:
We have the [`privacy_request_status_pending` fixture](https://github.com/ethyca/fidesops/blob/main/tests/ops/fixtures/application_fixtures.py#L800-L808) for this

Reply:
Nice, I wasn't aware of that. Updated now